src/bidiMapper/domains/context/CdpTarget.ts

```diff
@@ -40,6 +41,7 @@ export class CdpTarget {
 
   readonly #targetUnblocked = new Deferred<Result<void>>();
   readonly #acceptInsecureCerts: boolean;
+  #networkDomainEnabled = false;
```

Comment:
Maybe make it deferred?

Reply:
Easier concurrency handling

Reply:
We can put concurrent enabling/disablling commands in a queue which will reduce a chance of race condition.

Reply:
This suggestion is optional though